registry/storage/driver/s3-aws/s3.go

```diff
@@ -427,6 +446,10 @@ func New(params DriverParameters) (*Driver, error) {
 		awsConfig.WithEndpoint(params.RegionEndpoint)
 	}
 
+	if params.S3Accelerate {
```

Comment:
Minor comment: just use `awsConfig.WithS3UseAccelerate(params.S3Accelerate)`？It can be concise.